behave_test_automation/pah_behave/features/page_objects/new_drive_page.py

```diff
@@ -25,25 +29,29 @@ def choose_dropdown_option(self, name):
     def error_add_new_drive(self, name):
         return By.XPATH, f'//li[contains(text(), "{name} is required")]'
 
-    def submit_add_new_drive_required_fields(self, start_location, starting_mileage, project, car, passenger,
+    def first_select_option(self, select_name):
+        first_option = Select(self.browser.find_element_by_name(f'{select_name}'))
```

Comment:
```suggestion
        first_option = Select(self.browser.find_element_by_name(select_name))
```
?

Reply:
you can use `str()` if it _has_ to be a string, but I don't get why passing `select_name` openly won't work.

Reply:
Passing `select_name` will work, I made a rookie mistake :) Changes applied.